lib/custom/setup/TablesMigrateSiteidLaravel.php

```diff
@@ -54,6 +55,9 @@ public function migrate()
 		$this->process( $this->resources );
 
 		if( $this->getSchema( 'db-customer' )->tableExists( 'users' ) !== false ) {
```

Comment:
Thank you for your pull request but the solution is simply to change:
```
->tableExists( 'users' )
```
to
```
->columnExists( 'users', 'siteid' )
```
Can you change that?

Reply:
Done. Thank you!